[Home](https://andre-ye.github.io) > [Biology Navigation](https://andre-ye.github.io/biology/biology_navigation) > [Biology Content](https://andre-ye.github.io/biology/biology_navigation#biology-content) > Genes and DNA > Textbook Notes - "Transcription, RNA Processing, and Translation" - Sections 17.1, 17.2, 17.3, 17.4, 17.5

# Textbook Notes - "Transcription, RNA Processing, and Translation" - Sections 17.1, 17.2, 17.3, 17.4, 17.5
Andre Ye, 11/17/2020

---

## 17.1: An Overview of Transcription
- The first step in using genetic information is transcription.
  - This is the synthesis of an RNA version of the instructions in archive.
- Enzymes called **RNA polymerases** are important in transcription.
  - RNA polymerases use monomers called ribonucleotide triphosphates (dNTPs) used for DNA synthesis.
- These are like dNTPs used for DNA synthesis but have a hydroxyl group at the 2' carbon.
- Once an NTP is in place, the RNA polymerase catalyzes a reaction that cleaves off the two phosphates and forms a phosphodiester linkage.
  - Forms between the 3' end of the growing mRNA chain and the new ribonucleoside monophosphate.
- RNA that is complementary to one of the DNA strands is synthesized in 5' to 3' direction.
- Terminology:
  - The strand read by RNA polymerase is the **template strand.**
  - The other strand is the **coding strand.**
    - The RNA sequence matches it, because it is synthesized in the same direction.
- RNA polymerase does not require a primer to begin transcription.
- Bacteria have a single RNA polymerase; eukaryotes have at least three distinct types.

### Initiation: How Does Transcription Begin in Bacteria?
- One way of thinking about a gene: a stretch of DNA that is transcribed to produce a functional product for a cell.
  - How does RNA polymerase find genes?
- **Initiation** is key to control.
- **Core enzyme**, which transcribes the genes, needs **sigma** protein to bind to the core enzyme to recognize sites where transcription should begin.
  - Sites are called **promoters**.
- Bacterial RNA polymerase core enzyme and sigma form a **holoenzyme** ('whole enzyme').

#### Bacterial Promoters
- David Pribnow: offered an initial answer in the mid-1970s.
- Pribnow found that promoters were 40-50 base pairs long and had a series of bases identical or similar to TATAAT.
  - Known as the -10 box, because it is centered about 10 bases from the point where transcriptionb egins.
- RNA polymerase moves *downstream* from a point of reference, DNA is said to be *upstream* from the point of reference.
- The sequence TTGACA is 35 bases upstream of the +1 site called the -35 box.

#### Events Inside the Holoenzyme
- Transcription can only be initiated when sigma binds to the -35 and -10 boxes in DNA.
- The sigma protein can bind the promoter in only one orientation.
- The orientation of the promoter determines which DNA strand will be used as the template and which direction RNA polymerase will start synthesizing.
- When the holoenzyme is bound to a promoter,
  1. RNA polymerase opens the DNA helix.
  2. DNA strands form a 'transcription bubble'.
- Ribonucleoside triphosphates (NTPs) enter a channel in the enzyme and diffuse to the active site.

### Elongation and Termination in Bacteria
- When the RNA polymerase leaves the promoter region as it synthesizes RNA, the **elongation** phase is under way.
- During elongation, the enzyme reads the DNA template on the 3' end at a rate of 50 nucleotides per second.
- RNA polymerase proofreads and corrects errors.
  - A macromolecular machine with a structure critical for its function.
- Differerent parts of the enzyme helped steer the template through channels inside the enzyme and to separate newly synthesized RNA from the DNA template.
- Double-stranded DNA goes in and out of one grove.
  - NTPs enter another one.
  - RNA strand exits out the rear.
- **Termination** ends transcription.
  - Transcription stops in bacteria when RNA polymerase transcribes a DNA sequence called a transcription-termination signal.

### Transcription in Eukaryotes
- Eukaryotes are somewhat similar in features of transcription to bacteria.
- Important differences:
  - Eukaryotes have **three** major polymerases.
    - RNA polymerase I, II, & III (pol I, pol II, pol III).
    - Each polymerase produces *certain types of RNA*. 
  - Promoters in eukaryotic DNA are larger and  more diverse.
    - Most eukaryotic promoters include a 'TATA box', centered 30 base pairs upstream of the transcription start site.
  - Eukaryotic RNA polymerases recognize promoters using **general transcription factors.**
    - First to assemble at the promoter, RNA polymerase follows.
  - Termination of transcription differs from termination in bacteria.
    - DNA sequence near the end of each gene called the polyadenylation signal (**poly(A) signal**) is transcribed.
    - RNA downstream continues to transcribe the DNA template.
    - RNA polymerase comes off the DNA template.
  - Transcription and translation are separated in time and space. 
    - Transcription occurs in the nucleus and translation occurs in ribosomes of the cytoplasm for eukaryotes.
    - Bacteria begins translating mRNA before its transcription is complete.
    
### Process: Initiating Transcription in Bacteria
1. Initiation begins. Sigma binds to the promoter region of the DNA, characterized by the -35 box and the -10 box.
2. Initiation continues. RNA polymerase opens up the DNA helix and transcription begins. NTPs are used to create the complement of the template strand. 
3. Initiation completes. Sigma is released from the core enzyme and RNA synthesis continues. The RNA polymerase moves downstream along the DNA.

### Process: Ending Transcription in Bacteria
1. Hairpin forms. RNA polymerase transcribes a transcription-termination signal, which codes for RNA that forms a hairpin. 
2. Termination. The RNA hairpin leads to the RNA separating from RNA polymerase, terminating transcription.

## 17.2: RNA Processing in Eukaryotes
- Newly transcribed eukaryotic RNAs were nonfunctional and many times larger than corresponding RNAs.
- Eukaryotic genes are initially copied from nonfunctional RNAs called **primary transcripts**.
  - Primary transcript is pre-mRNA for protein-coding genes.
- Primary transcript required **RNA processing** to generate mature and functional RNA.

### The Startling Discovery of Split Eukaryotic Genes
- Richard Roberts and Phillip Sharp (1977) discovered a common-cold virus had protein-coding genes with intervening sequences of noncodingDNA.
  - Key information *was split into pieces.*
- Experiment:
  1. Heat the virus's DNA to break hydrogen bonds.
  2. Single-stranded DNA incubated with mRNA.
    - Intention: promote base pairing to reveal where genes occurred.
- Parts of the DNA formed loops.
  - Stretches of DNA not represented in corresponding mRNA.
- Example: eukaryotic genes do not carry messages like
```
Biology is my favorite course of all time.
```
- Instead, the message would be something like
```
BIOLηεπpoενχνσoφγενεσOGY IS MY
FAVORαpεLντεppeπITε3vνoν中o命ωγ命ITECOURSE
OFαν{r可αωεTOl3εσπλLXTOγεTηεp ALL TIME
```
- Sections of a noncoding sequence are represented with random characters.
  - Must be removed from the RNA before it can carry an understandable message.

#### Introns and Exons
- Regions of a gene that are transcribed but not represented in the final RNA are introns.
- Regions that are transcribed and represented in final RNA are called exons (*ex*pressed in mature DNA).
- "intron" and "exon" also used for regions of transcribed RNA that are cut out (introns) or retained (exons).
- Exons are not necessarily protein-coding regions.
  - Many are, but some exons are RNA sequences that do not code for proteins.

### RNA Splicing
- Transcription of eukaryotic genes generates a primary transcript with exons and introns.
- Through **splicing**, introns are removed from the growing RNA strand.
  - Pieces of the primary transcript are removed and remaining segments are joined.
  - Results in uninterrupted RNA mesage.
- Protein + RNA macromolecular machiens are small nuclear ribonucleoproteins (snRNPs).

#### The Process of RNA Splicing
1. snRNPs (small nuclear ribonucleoproteins) bind to the start and end of an intron and to a branch site within the intron.
2. More snRNPs join, and a spliceosome assembles. These are large macromolecular machines that form as an aggregate of snRNPs.
3. The 5' end of the intron is cut from the exon, and the intron forms a single-stranded stem with a loop (a lariat) with an adenine in its connecting branch point.
4. The 3' end of the intron is cut, releasing the intron as a lariat. Exons are joined by a phosphodiester linkage. Cut intron is degraded to ribonucleoside monophosphates.

### Adding Caps and Tails to Transcripts
- Two additional processing events are required for splicing.
  1. When 5' end of pre-mRNA emerges, enzymes add a 5' cap.
    - Modified guanine nucleotide linked to transcript in an unusual way.
    - Enables ribosomes to bind to the mRNA and protects the 5' end of the mRNA from enzymes that degrade RNA (ribonucleases).
  2. Enzyme cuts the 3' end of the pre-MRNA after the poly(A) signal.
    - Specialized RNA polymerase adds 100 to 250 adenine nucleotides.
    - Known as the poly(A) tail.
    - Not encoded by the DNA template strand.
    - Required for ribosomes to start translation and to protect the end of mRNA from attack by enzymes.






