[Home](https://andre-ye.github.io) > [Biology Navigation](https://andre-ye.github.io/biology/biology_navigation) > [Biology Content](https://andre-ye.github.io/biology/biology_navigation#biology-content) > Genes and DNA > Textbook Notes - "Transcription, RNA Processing, and Translation" - Sections 17.1, 17.2, 17.3, 17.4, 17.5

# Textbook Notes - "Transcription, RNA Processing, and Translation" - Sections 17.1, 17.2, 17.3, 17.4, 17.5
Andre Ye, 11/17/2020

---

## Navigate
- [17.1: An Overview of Transcription](#171-an-overview-of-transcription)
  * [Initiation: How Does Transcription Begin in Bacteria?](#initiation-how-does-transcription-begin-in-bacteria)
    + [Bacterial Promoters](#bacterial-promoters)
    + [Events Inside the Holoenzyme](#events-inside-the-holoenzyme)
  * [Elongation and Termination in Bacteria](#elongation-and-termination-in-bacteria)
  * [Transcription in Eukaryotes](#transcription-in-eukaryotes)
  * [Process: Initiating Transcription in Bacteria](#process-initiating-transcription-in-bacteria)
  * [Process: Ending Transcription in Bacteria](#process-ending-transcription-in-bacteria)
- [17.2: RNA Processing in Eukaryotes](#172-rna-processing-in-eukaryotes)
  * [The Startling Discovery of Split Eukaryotic Genes](#the-startling-discovery-of-split-eukaryotic-genes)
    + [Introns and Exons](#introns-and-exons)
  * [RNA Splicing](#rna-splicing)
    + [The Process of RNA Splicing](#the-process-of-rna-splicing)
  * [Adding Caps and Tails to Transcripts](#adding-caps-and-tails-to-transcripts)
- [17.3: An Introduction to Translation](#173-an-introduction-to-translation)
  * [Ribosomes Are the Site of Protein Synthesis](#ribosomes-are-the-site-of-protein-synthesis)
  * [An Overview of Translation](#an-overview-of-translation)
  * [How Does mRNA Specify Amino Acids?](#how-does-mrna-specify-amino-acids)
    + [Two Hypotheses of Codon Interactions with Amino Acids](#two-hypotheses-of-codon-interactions-with-amino-acids)
- [17.4: The Structure and Function of Transfer RNA](#174-the-structure-and-function-of-transfer-rna)
  * [Zamecnik Experiment](#zamecnik-experiment)
  * [What is the Structure of tRNAs?](#what-is-the-structure-of-trnas)
  * [How Are Amino Acids Attached to tRNAs?](#how-are-amino-acids-attached-to-trnas)
  * [How Many Types of tRNAs Are There?](#how-many-types-of-trnas-are-there)
- [17.5: Ribosome Structure and Function in Translation](#175-ribosome-structure-and-function-in-translation)
  * [Initiating Translation](#initiating-translation)
    + [Process of Initiating Translation in Bacteria](#process-of-initiating-translation-in-bacteria)
  * [Elongation: Extending the Polypeptide](#elongation-extending-the-polypeptide)
    + [Is the Ribosome and Enzyme or a Ribozyme?](#is-the-ribosome-and-enzyme-or-a-ribozyme)
    + [Moving Down the mRNA](#moving-down-the-mrna)
    + [Process of Elongation Phase of Translation](#process-of-elongation-phase-of-translation)
  * [Terminating Translation](#terminating-translation)
    + [Process of Terminating Translation](#process-of-terminating-translation)
  * [Polypeptides Are Modified After Translation](#polypeptides-are-modified-after-translation)
    + [Polypeptide Folding](#polypeptide-folding)
    + [Chemical Modifications](#chemical-modifications)

## 17.1: An Overview of Transcription
- The first step in using genetic information is transcription.
  - This is the synthesis of an RNA version of the instructions in archive.
- Enzymes called **RNA polymerases** are important in transcription.
  - RNA polymerases use monomers called ribonucleotide triphosphates (dNTPs) used for DNA synthesis.
- These are like dNTPs used for DNA synthesis but have a hydroxyl group at the 2' carbon.
- Once an NTP is in place, the RNA polymerase catalyzes a reaction that cleaves off the two phosphates and forms a phosphodiester linkage.
  - Forms between the 3' end of the growing mRNA chain and the new ribonucleoside monophosphate.
- RNA that is complementary to one of the DNA strands is synthesized in 5' to 3' direction.
- Terminology:
  - The strand read by RNA polymerase is the **template strand.**
  - The other strand is the **coding strand.**
    - The RNA sequence matches it, because it is synthesized in the same direction.
- RNA polymerase does not require a primer to begin transcription.
- Bacteria have a single RNA polymerase; eukaryotes have at least three distinct types.

### Initiation: How Does Transcription Begin in Bacteria?
- One way of thinking about a gene: a stretch of DNA that is transcribed to produce a functional product for a cell.
  - How does RNA polymerase find genes?
- **Initiation** is key to control.
- **Core enzyme**, which transcribes the genes, needs **sigma** protein to bind to the core enzyme to recognize sites where transcription should begin.
  - Sites are called **promoters**.
- Bacterial RNA polymerase core enzyme and sigma form a **holoenzyme** ('whole enzyme').

#### Bacterial Promoters
- David Pribnow: offered an initial answer in the mid-1970s.
- Pribnow found that promoters were 40-50 base pairs long and had a series of bases identical or similar to TATAAT.
  - Known as the -10 box, because it is centered about 10 bases from the point where transcriptionb egins.
- RNA polymerase moves *downstream* from a point of reference, DNA is said to be *upstream* from the point of reference.
- The sequence TTGACA is 35 bases upstream of the +1 site called the -35 box.

#### Events Inside the Holoenzyme
- Transcription can only be initiated when sigma binds to the -35 and -10 boxes in DNA.
- The sigma protein can bind the promoter in only one orientation.
- The orientation of the promoter determines which DNA strand will be used as the template and which direction RNA polymerase will start synthesizing.
- When the holoenzyme is bound to a promoter,
  1. RNA polymerase opens the DNA helix.
  2. DNA strands form a 'transcription bubble'.
- Ribonucleoside triphosphates (NTPs) enter a channel in the enzyme and diffuse to the active site.

### Elongation and Termination in Bacteria
- When the RNA polymerase leaves the promoter region as it synthesizes RNA, the **elongation** phase is under way.
- During elongation, the enzyme reads the DNA template on the 3' end at a rate of 50 nucleotides per second.
- RNA polymerase proofreads and corrects errors.
  - A macromolecular machine with a structure critical for its function.
- Differerent parts of the enzyme helped steer the template through channels inside the enzyme and to separate newly synthesized RNA from the DNA template.
- Double-stranded DNA goes in and out of one grove.
  - NTPs enter another one.
  - RNA strand exits out the rear.
- **Termination** ends transcription.
  - Transcription stops in bacteria when RNA polymerase transcribes a DNA sequence called a transcription-termination signal.

### Transcription in Eukaryotes
- Eukaryotes are somewhat similar in features of transcription to bacteria.
- Important differences:
  - Eukaryotes have **three** major polymerases.
    - RNA polymerase I, II, & III (pol I, pol II, pol III).
    - Each polymerase produces *certain types of RNA*. 
  - Promoters in eukaryotic DNA are larger and  more diverse.
    - Most eukaryotic promoters include a 'TATA box', centered 30 base pairs upstream of the transcription start site.
  - Eukaryotic RNA polymerases recognize promoters using **general transcription factors.**
    - First to assemble at the promoter, RNA polymerase follows.
  - Termination of transcription differs from termination in bacteria.
    - DNA sequence near the end of each gene called the polyadenylation signal (**poly(A) signal**) is transcribed.
    - RNA downstream continues to transcribe the DNA template.
    - RNA polymerase comes off the DNA template.
  - Transcription and translation are separated in time and space. 
    - Transcription occurs in the nucleus and translation occurs in ribosomes of the cytoplasm for eukaryotes.
    - Bacteria begins translating mRNA before its transcription is complete.
    
### Process: Initiating Transcription in Bacteria
1. Initiation begins. Sigma binds to the promoter region of the DNA, characterized by the -35 box and the -10 box.
2. Initiation continues. RNA polymerase opens up the DNA helix and transcription begins. NTPs are used to create the complement of the template strand. 
3. Initiation completes. Sigma is released from the core enzyme and RNA synthesis continues. The RNA polymerase moves downstream along the DNA.

### Process: Ending Transcription in Bacteria
1. Hairpin forms. RNA polymerase transcribes a transcription-termination signal, which codes for RNA that forms a hairpin. 
2. Termination. The RNA hairpin leads to the RNA separating from RNA polymerase, terminating transcription.

## 17.2: RNA Processing in Eukaryotes
- Newly transcribed eukaryotic RNAs were nonfunctional and many times larger than corresponding RNAs.
- Eukaryotic genes are initially copied from nonfunctional RNAs called **primary transcripts**.
  - Primary transcript is pre-mRNA for protein-coding genes.
- Primary transcript required **RNA processing** to generate mature and functional RNA.

### The Startling Discovery of Split Eukaryotic Genes
- Richard Roberts and Phillip Sharp (1977) discovered a common-cold virus had protein-coding genes with intervening sequences of noncodingDNA.
  - Key information *was split into pieces.*
- Experiment:
  1. Heat the virus's DNA to break hydrogen bonds.
  2. Single-stranded DNA incubated with mRNA.
    - Intention: promote base pairing to reveal where genes occurred.
- Parts of the DNA formed loops.
  - Stretches of DNA not represented in corresponding mRNA.
- Example: eukaryotic genes do not carry messages like
```
Biology is my favorite course of all time.
```
- Instead, the message would be something like
```
BIOLηεπpoενχνσoφγενεσOGY IS MY
FAVORαpεLντεppeπITε3vνoν中o命ωγ命ITECOURSE
OFαν{r可αωεTOl3εσπλLXTOγεTηεp ALL TIME
```
- Sections of a noncoding sequence are represented with random characters.
  - Must be removed from the RNA before it can carry an understandable message.

#### Introns and Exons
- Regions of a gene that are transcribed but not represented in the final RNA are introns.
- Regions that are transcribed and represented in final RNA are called exons (*ex*pressed in mature DNA).
- "intron" and "exon" also used for regions of transcribed RNA that are cut out (introns) or retained (exons).
- Exons are not necessarily protein-coding regions.
  - Many are, but some exons are RNA sequences that do not code for proteins.

### RNA Splicing
- Transcription of eukaryotic genes generates a primary transcript with exons and introns.
- Through **splicing**, introns are removed from the growing RNA strand.
  - Pieces of the primary transcript are removed and remaining segments are joined.
  - Results in uninterrupted RNA mesage.
- Protein + RNA macromolecular machiens are small nuclear ribonucleoproteins (snRNPs).

#### The Process of RNA Splicing
1. snRNPs (small nuclear ribonucleoproteins) bind to the start and end of an intron and to a branch site within the intron.
2. More snRNPs join, and a spliceosome assembles. These are large macromolecular machines that form as an aggregate of snRNPs.
3. The 5' end of the intron is cut from the exon, and the intron forms a single-stranded stem with a loop (a lariat) with an adenine in its connecting branch point.
4. The 3' end of the intron is cut, releasing the intron as a lariat. Exons are joined by a phosphodiester linkage. Cut intron is degraded to ribonucleoside monophosphates.

### Adding Caps and Tails to Transcripts
- Two additional processing events are required for splicing.
  1. When 5' end of pre-mRNA emerges, enzymes add a 5' cap.
    - Modified guanine nucleotide linked to transcript in an unusual way.
    - Enables ribosomes to bind to the mRNA and protects the 5' end of the mRNA from enzymes that degrade RNA (ribonucleases).
  2. Enzyme cuts the 3' end of the pre-MRNA after the poly(A) signal.
    - Specialized RNA polymerase adds 100 to 250 adenine nucleotides.
    - Known as the poly(A) tail.
    - Not encoded by the DNA template strand.
    - Required for ribosomes to start translation and to protect the end of mRNA from attack by enzymes.

![](https://andre-ye.github.io/biology/genes-and-dna/pre%20mrna%20diagram.PNG)

## 17.3: An Introduction to Translation
- To synthesize a protein, the sequence of bases in a messenger RNA molecule must be translated into a sequence of amino acids in a polypeptide.

### Ribosomes Are the Site of Protein Synthesis
- *Where does translation occur?*
- Answer: observation that there is correlation between number of ribosomes in a cell and rate of protein synthesis.
  - Test: Roy Britten et al. performed a pulse-chase experiment.
  - Concluded that proteins are synthesized at ribosomes and then released.

### An Overview of Translation
- Decade after ribosome hypothesis confirmed: electron micrographs showed bacterial ribosomes in action.
  - Bacteria: ribosomes can attach to mRNA and begin synthesizing proteins before transcription was complete.
  - Multiple ribosomes attach to each mRNA.
    - Structure of this is a polyribosome.
  - Polyribosomes increase the number of copies of a protein that can be made from single mRNA.
- Transcription and translation can be coupled in bacteria because there is no nuclear envelope to separate the two processes.
  - Gene expression can hence be very fast.
- Eukaryotes, transcription and translation are separated by *time and space.*
  - When mRNAs are outside of the nucleus, ribosomes can attach and begin translation.

  
### How Does mRNA Specify Amino Acids?
- Discovery of genetic code revealed that triplet codons in mRNA specify particular amino acids.
  - How does this conversion happen?
  
#### Two Hypotheses of Codon Interactions with Amino Acids
- **Hypothesis 1:** Bases in a particular codon were complementary in shape or charge to the side group of an amino acid.
  - Problem: how could nucleic acid bases itneract with nonpolar amino acid side groups?
- **Hypothesis 2:** Adapter molecules hold amino acids in place while interacting directly.

## 17.4: The Structure and Function of Transfer RNA
- Crick's ideas for an adapter molecule were discovered by accident.
- Ribosomes provide the catalytic machinery, mRNAs contribute the message to be translated, amino acids are the building blocks of proteins, and ATP supplies energy.
  - **Not enough.** Transfer RNA (tRNA) was needed to complete the picture.
- tRNA has an amino acid attached, known as **aminoacyl tRNA**.

### Zamecnik Experiment
- What happens to amino acids bound to tRNAs?
- Paul Zamecnik et al tracked radioactive leucine molecules attached to tRNAs; found that amino acids were transferred from tRNAs to proteins.

| Aspect | Description |
| --- | --- |
| Question | What happens to amino acids attached to tRNAs? |
| Hypothesis | Aminoacyl tRNAs transfer amino acids to growing polypeptides. |
| Null Hypothesis | Aminoacyl tRNAs do not transfer amino acids to growing polypeptides. |
| Experimental Setup | Attach radioactive leucine molecules to tRNAs. Attach aminoacyl tRNAs to an in vitro system that allows protein synthesis. Follow the path of radioactive amino acids. |
| Prediction of Hypothesis | Radioactive amino acids will be found in the polypeptides. |
| Prediction of Null Hypothesis | Radioactive amino acids will not be found in the polypeptides. |
| Results | Radioactive signal of tRNA and polypeptides are inversely proportional to each other. When there is low radioactive signal in tRNA, there is high radioactive signal in the polypeptides, and vice versa. |
| Conclusion | Aminoacyl tRNAs transfer amino acids to growing polypeptides. |

### What is the Structure of tRNAs?
- tRNA serves as a chemical go-between that allows amino acids to interact with an mRNA template.
- Initial studies established the sequence of nucleotides in tRNAs.
  - Usually short (75-95 nucleotides).
  - Certain parts of each tRNA could form hydrogen bonds with complementary base sequences somewhere else in the same molecule.
    - Forms stem-and-loop structures.
- **Two important parts of tRNA:**
  - CCA sequence at 3' end of tRNA molecule is the site for amino acid attachment.
  - Loop opposite to the amino acid attachment site contains three ribonucleotides that serve as an **anticodon.**
    - Anticodon is a triplet of ribonucleotides that can form base pairs with the codon for the amino acid in mRNA. e.g. anticodon os AAA is UUU.
- X-ray crystallography revealed tertiary structure of tRNA; folds into an L-shaped molecule with an anticodon on one end and the CCA sequence + attached amino acid on another end.

### How Are Amino Acids Attached to tRNAs?
- Three important components of linking amino acids to tRNAs:
  - Input of energy from ATP required to attach amino acid to tRNA.
  - Enzymes called **aminoacyl-tRNA synthetates** catalyze the addition of amino acids to tRNAs. Known as 'charging' a tRNA.
  - For each of the 20 major amino acids, there is a different aminoacyl-tRNA synthetase and one or more tRNAs.
- Each synthetase has a binding site for one amino acid and tRNA.
  - Differences in tRNA shape let enzymes recognize and match tRNA to the amino acid.
- The tRNA and aminoa cid structures fit together.
  - Precision is important; frequent mistakes in choosing the matching amino acid are disasterous.

### How Many Types of tRNAs Are There?
- Paradox: genetic code specifies 20 common amino acids using 61 different codons.
  - There should be 61 different tRNAs to read codons.
  - Most cells have about 40.
- How can 61 codons be translated with 40 tRNAs?
- **Wobble hypothesis:**
  - (Most) Amino acids are specified by more than one codon.
  - Codons for the same amino acid often have the same nucleotides for the first and second positions but different nucleotides in the third position.
  - `CAA` and `CAG` both code for glutamine; tRNA with GUU anticodon can pair with the first two bases of both.
  - tRNA reads the codon because the `U` in anticodon third position can form a nonstandard base pair with `G` in `CAG`.
  - Crick: proposed that certain bases bind to bases that do not match Watson-Crick base pairing.
    - Flexibility 'wobble' in base pairing.
  - Wobble pairing allows one tRNA to read more than one codon.
- Wobbles in the third position of a codon explain 40 tRNAs translating 61 arrangements of codons.
- **Wobble pairing does not account for redundancy**, it only explains how on tRNA can read more than one codon, not how one amino acid can be specified by more than one codon.

## 17.5: Ribosome Structure and Function in Translation
- Translation of each codon in mRNA into the next amino acid *begins when the anticodon of tRNA binds to the codon.*
- Codon translation is complete when a peptide bond forms between the amino acid and the polypeptide.
- rRNAs (ribosomal RNAQs) that, with proteins, make up ribosomes.
- Ribosomes separate into a large and small subunit.
  - Each ribosome consists of many rRNA molecules and proteins.
  - **Small subunit hold mRNA, large subunit is where peptide bonds are fromed.**
- Three sites:
  - tRNA carries an amino acid. This site is the 'A' site (acceptor or aminoacyl).
  - tRNA in the middle holds the growing polypeptide chain and occupies the P (peptidyl) site in the ribosome. Think as 'P' for peptide-bond formation.
  - tRNA that no longer has amino acids attached leave the ribosome, occupying the E site ('E' for exit).
- Process of protein synthesis in ribosomes:
  1. Aminoacyl tRNA diffuses into the A site. If its anticodon matches a codon in mRNA, it stays in the ribosome.
  2. A peptide bond forms between the amino acid held by the aminoacyl tRNA in the A site and the growing polypeptide in the P site.
  3. The ribosome moves relative to the mRNA by one codon. All three tRNAs are shifted one position within the ribosome.
    - tRNA in E site exits.
    - tRNA in P site moves to the E site.
    - tRNA in A site switches to P site.
    - A site is empty and ready to accept another aminoacyl tRNA.
- Protein being synthesized grows by one amino acid each time.
  - Protein synthesis begins at the amino end (N-terminus) of the polypeptide and proceeds towards the carboxy end (C-terminus).
  
### Initiating Translation
- To translate an mRNA, a ribosome must begin at the first codon in a message, translate the mRNA up to the termination, then stop.
- Three phrases of protein synthesis: initiation, elongation, and termination.
- Start codon is found near the 5' end of mRNA; codes for amino acid methionine.
  - Distinguish initiation of translation at the start codon from initiation of transcription at the promoter.
- Translation *begins when a **section of rRNA in a small ribosomal subnit binds to a complementary sequence on an mRNA.***
  - This mRNA region is the **ribosome binding site** (also called the **Shine-Dalgarno sequence**).
  - About six nucleotides upstream from the start codon.
- Interactions betweenthe small subunit, message, and tRNA are mediated by **initiation factors**.
  - Help prepare the ribosome for translation and in binding the first aminoacyl tRNA.
  - Modified form of methionine - N-formylmethionine (abbreviated f-Met).
- In **eukaryotes**:
  - More initiation factors are needed.
  - Ribosome first associates with cap on the 5' end of mRNA.
  - Initiator tRNA carries normal methionine.
- Initiation is complete when the large subunit joins the complex.
  - Initiator tRNA occupies the P site. This is the only time a tRNA carrying a single amino acid occupies the P site.
- Summary:
  - Translation initiation in bacteria is three-pronged.
    1. mRNA binds to a small ribosomal subunit.
    2. Initiator tRNA with f-Met binds to the start codon.
    3. Large ribosomal subunit binds.
  - Initiation in eukaryotes is different but has similarities.

#### Process of Initiating Translation in Bacteria
1. **mRNA binds to a small subunit.** A sequence in mRNA called the ribosome binding site binds to a complementary seuqnece in an RNA molecule, which is part of the small subunit of the ribosome. This is helped by initiation factors.
2. **Initiator aminoacyl tRNA binds to the start codon.** This usually carries f-Met.
3. The **large subuit of the ribosome binds**, completing the ribosome assembly; translation can begin.

### Elongation: Extending the Polypeptide
- At the start of elongation, the E and A sites in the ribosome are empty of tRNAs.
- An mRNA codon is exposed in the A site.
- The elongation phase begins when an aminoacyl tRNA binds to the codon in the A site.
- See process for more details on the elongation phase.

#### Is the Ribosome and Enzyme or a Ribozyme?
- Ribosomes contain roughly equal amounts of protein and RNA.
- The active site consisted entirely of RNA; therefore the ribosome is a ribozyme (not a protein-based enzyme).

#### Moving Down the mRNA
- Peptide bond formation involves the transfer of the amino acid linked to the tRNA in the P site to the amino acid held by the tRNA in the A site.
- During **translocation**, the ribosome moves one codon down from the mRNA once a new peptide bond is formed.
  - An mRNA in reality is ratcheted througha  stationary ribosome.
  - Important point: translocation is a codon-by-codon movement.
_ **Elongation factors** are proteins required for translocation.
  - Translocation requires energy; elongation factors bind to the ribosome and break down GTP, an energy-rich molecule.
- Anticodons of tRNA are bound to the codons of mRNA; movement of ribosome brings uncharged tRNA into the E site and the tRNA containing the growing polypeptide in the P site.

#### Process of Elongation Phase of Translation
1. **Incoming aminoacyl tRNA** moves into the A site. Its anticodon base pairs with the mRNA codon.
2. **Peptide bond formation**. The amino acid attached to the tRNA in the P site is transferred by formation of a peptide bond to the amino acid of the tRNA in the A site.
3. **Translocation**. The ribosome moves one codon down the mRNA with the help of elongation factors. THe tRNA attached to the polypeptide moves into the P site. The A site is empty.
4. **Incoming aminoacyl tRNA**. A new charged tRNA moves into the A site, where its anticodon base-pairs with an mRNA codon.
5. **Peptide-bond formation.** The polypeptide chain attached to the tRNA in the P site is transferred by peptide bond formation to the aminoacyl tRNA in the A site.
6. **Translocation.** THe ribosome moves one codon down the mRNA. The tRNA attached to the polypeptide chain moves into the P site. Uncharged tRNA from P site moves to the E site, where tRNA is ejected. The A site becomes empty again.

### Terminating Translation
- Genetic code includes three stop codons - `UAA`, `UAG`, and `UGA`.
- tRNAs do not work to terminate translation.
  - The translocating ribosome reaches one of the stop codons and a protein called a **release factor** recognizes the stop codon and fills the A site.
- Stop codons found in the 3' region of mRNA.
- Release factors fit tightly into the A site; they have the size and shape of an aminoacyl tRNA.
  - Once in the A site, release factor triggers the hydrolysis of the bond that links tRNA in the P site and frees the polypeptide.
  
#### Process of Terminating Translation
1. **Release factor binds to stop codon.** When the translocating ribosome reaches a stop codon, a protein release factor fills up the A site and breaks the bond linking tRNA to the polypeptide chain.
2. **Polypeptide and uncharged tRNA are released.**
3. **Ribosome subunits separate.** They are ready to attach tot he start codon of another message.

### Polypeptides Are Modified After Translation
- Proteins are not fully formed or functional at the end of translation.

#### Polypeptide Folding
- A protein's function depends on its shape, which depends on how it folds.
- Folding is guided and accelerated by proteins called molecular chaperones.

#### Chemical Modifications
- Eukaryotic proteins are extensively modified even after they are synthesized.
  - e.g. addition of sugars, lipid groups, phosphate groups.

[Back to top](#)
